main.go

```diff
@@ -43,34 +45,34 @@ const (
 
 var (
 	defaultCollectors = collectorSet{
-		"daemonsets":             struct{}{},
-		"deployments":            struct{}{},
-		"limitranges":            struct{}{},
-		"nodes":                  struct{}{},
-		"pods":                   struct{}{},
-		"replicasets":            struct{}{},
-		"replicationcontrollers": struct{}{},
-		"resourcequotas":         struct{}{},
-		"services":               struct{}{},
-		"jobs":                   struct{}{},
-		"cronjobs":               struct{}{},
-		"statefulsets":           struct{}{},
-		"persistentvolumeclaims": struct{}{},
+		"daemonsets/v1beta1":        struct{}{},
+		"deployments/v1beta1":       struct{}{},
+		"limitranges/v1":            struct{}{},
+		"nodes/v1":                  struct{}{},
+		"pods/v1":                   struct{}{},
+		"replicasets/v1beta1":       struct{}{},
+		"replicationcontrollers/v1": struct{}{},
+		"resourcequotas/v1":         struct{}{},
+		"services/v1":               struct{}{},
+		"jobs/v1":                   struct{}{},
+		"cronjobs/v2alpha1":         struct{}{},
+		"statefulsets/v1beta1":      struct{}{},
+		"persistentvolumeclaims/v1": struct{}{},
 	}
 	availableCollectors = map[string]func(registry prometheus.Registerer, kubeClient clientset.Interface){
-		"cronjobs":               collectors.RegisterCronJobCollector,
-		"daemonsets":             collectors.RegisterDaemonSetCollector,
-		"deployments":            collectors.RegisterDeploymentCollector,
-		"jobs":                   collectors.RegisterJobCollector,
-		"limitranges":            collectors.RegisterLimitRangeCollector,
-		"nodes":                  collectors.RegisterNodeCollector,
-		"pods":                   collectors.RegisterPodCollector,
-		"replicasets":            collectors.RegisterReplicaSetCollector,
-		"replicationcontrollers": collectors.RegisterReplicationControllerCollector,
-		"resourcequotas":         collectors.RegisterResourceQuotaCollector,
-		"services":               collectors.RegisterServiceCollector,
-		"statefulsets":           collectors.RegisterStatefulSetCollector,
-		"persistentvolumeclaims": collectors.RegisterPersistentVolumeClaimCollector,
+		"cronjobs/v2alpha1":         collectors.RegisterCronJobCollector,
+		"daemonsets/v1beta1":        collectors.RegisterDaemonSetCollector,
```

Comment:
I have restored to the original state, used `variable` to shown the GroupVersionResource of collector.